angular-todo/e2e/manage-todo.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from '@playwright/test';
+import { v4 as uuid } from 'uuid';
+
+const toDo = uuid() + ' ' + 'TEST';
+const toDoEdited = uuid();
+
+test('manage todo angular', async ({ page }) => {
+  //create new todo
+  await page.goto('https://localhost:8443/home');
+  await page.getByRole('link', { name: 'Sign In', exact: true }).click();
+  await page.getByPlaceholder('User Name').fill('user01');
+  await page.getByPlaceholder('Password').fill('Password1!');
+  await page.getByPlaceholder('Password').press('Enter');
+  await page.getByRole('link', { name: 'Manage your todos here' }).click();
+  await page.getByPlaceholder('What needs doing?').click();
+  await page.getByPlaceholder('What needs doing?').fill(toDo);
+  await page.getByRole('button', { name: 'Create' }).click();
+  await expect(page.getByText(toDo)).toBeVisible();
+
+  //Edit todo
+  await page.locator('li').filter({ hasText: 'Edit Delete' }).locator('div').nth(1).click();
```

Comment:
Hmm, is there a more accessible way of doing this? I think the `filter` is good. I feel like `locater` may not be the best approach.

Reply:
Selectors like this are extremely brittle. We need to avoid this. Rather than writing a wild selector trying to find this element, can you add visually hidden text to the 3 dot button to the source code that opens this menu in the app? This way, it's more accessible to Playwright as well as visually challenged users.

Reply:
done